### PR TITLE
Feature 3

### PR DESCRIPTION
**Pull request by MANYIKAM**
Portfolio setup and mobile first

- Contact Details form has been added  with the following information;
- Description message 
- Full names of the individual
- Contact details inform of email
- Text area for any other desired information to be disclosed prior to submission
- Submission button at the end after capturing all the required fields on the form.

### CODE REVIEW

File: Style.css [hunk 1]
```diff
@@ -309,3 +309,55 @@ div {
...
+}
+
+.footer-cont {
+  background-color: #6070ff;
+  border-radius: 60px 0 0 0;
+  padding: 50px;
+  background-image: url('img/contactme.png');
+  background-repeat: no-repeat;
+  background-position: right;
+}
```
Comment (HAMMAS-SALEEM) [inline Style.css:327]: white-spacing-finish-mobile


- [ ] Kindly remove `padding: 50px;` to remove the white spacing in your design at the resolution of 375px :blush:

File: index.html [hunk 2]
```diff
@@ -134,5 +134,20 @@ <h3 class="resume">Languages<span class="yes">&#x2228</span></h3>
              </ul>
```
Comment (HAMMAS-SALEEM) [inline index.html:134]: contents
Screenshot from 2022-10-04 01-12-07


- [ ] Kindly make your contents align as you can see in the Figma to make your design professional :+1:

Comment (HAMMAS-SALEEM) [inline index.html:134]: ### Your Design
Screenshot from 2022-10-04 00-57-12
### Figma Template

center


Kindly make your contents align in center as you can see in the Figma template :+1: to make your design look professional :blush:

File: README.md [hunk 3]
```diff
@@ -1,10 +1,11 @@
 # Portfolio-setup-and-mobile-first
-Building the first 2 sections of the mobile website using the template I  chose from the previous  Steps
+Building the the Contact Details Section of the mobile first portfolio
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
 # Project Name
 
 > Portfolio-setup-and-mobile-first
+ -Contact Details section
```
Comment (HAMMAS-SALEEM) [inline README.md:11]: - [ ] You have added a link to your license file but didn't add a license file in your project. Kindly add an license file using these [guidlines](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository) to make your readme file descriptive :blush:

**Review by HAMMAS-SALEEM — CHANGES_REQUESTED** [review]
# Required Changes :recycle:
almost-there

## Hey @MANYIKAM :wave:

## Highlights :1st_place_medal:

:heavy_check_mark: Following GitHub Flow
:heavy_check_mark: Following some requirements
:heavy_check_mark: Meaningful commit messages
:heavy_check_mark: Descriptive PR Title
:heavy_check_mark: Descriptive PR Summary

Good job on implementing some of the requirements, there are some issues that you still need to work on to go to the next project but you are almost there! :blush:

# Optional

_All the changes with optional tags are not crucial enough to prevent you from getting the approval but we highly recommend to students that they should take these changes into consideration._

Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

---------------------------------------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Qoosim — APPROVED** [review]
#  *Status: Approved* :tada: :heavy_check_mark: 

Hi @MANYIKAM, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

_**Kindly check some optional comments I left under the review**_ :pray: 

## Highlights

-  No linter errros. :heavy_check_mark: 
-  Correct GitHub Flow. :heavy_check_mark: 
-  Meaningful commit messages. :heavy_check_mark: 



## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

- [OPTIONAL] Kindly look into the list of the tech you used to build a project on each of the projects. Only javascript tech displays. The rest are not showing. Please, look into it and fix it.

### Your Design

your-design-for-html-css-js

### Figma Design

figma-design-for-html-css-js

- [ ] Your language, frameworks and skills section broke. Maybe when you were trying to make changes required by the previous reviewer. Kindly look into that too and fix them.

### Your Design

your-lang


### Figma Design

figma-lang

**Comment by MANYIKAM** [comment]
Mobile first Set-up Portfolio has been completed in line to the provided information

- First Section comprises of the owners details and brief description
- Followed by portfolio projects done entitled Works , which shows almost all the the completed task with reference to years.
- Details of the languages specialized in and their sub divisions
- final the contact details with additional space to leave vital or message the owner.

**Pull request merged**